quora-api/src/main/java/com/upgrad/quora/api/controller/SignupController.java

```diff
@@ -0,0 +1,42 @@
+package com.upgrad.quora.api.controller;
+
+import com.upgrad.quora.api.model.SignupUserRequest;
+import com.upgrad.quora.api.model.SignupUserResponse;
+import com.upgrad.quora.service.business.SignupBusinessService;
+import com.upgrad.quora.service.entity.UserEntity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/")
+public class SignupController {
+
+    @Autowired
+    private SignupBusinessService signupBusinessService;
+    @RequestMapping(method = RequestMethod.POST, path = "/signup", consumes = MediaType.APPLICATION_JSON_UTF8_VALUE, produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
```

Comment:
Change the path = "/signup" to path = "/user/signup".  Please follow the guidelines for naming convention.

Reply:
Path has been updated to "/user/signup".